### ISSUE TITLE
Umbrella Issue for Accessibility Fixes 

### ISSUE THREAD

**Issue by dan-mba** [EddieHubCommunity/LinkFree#2382]
### Description

While adding accessibility tests in PR #2379 , the following issues failed the tests.

All pages:
- [x] Add lang attribute to html tag [instructions here](https://melvingeorge.me/blog/set-html-lang-attribute-in-nextjs) - #2415 
- [x] GitHub icon link in nav bar needs aria-label attribute - #2506 
- [x] Increase contrast ratio for "Powered by EddieHub" (current 2.53, required 4.5)- #2545 
- [x] Increase contrast ratio for "Donate with GitHub Sponsors" (current 2.53, required 4.5) - #2545 

Home Page 
- [x] Increase contrast ratio for "Users" purple box (current 2.98, required 4.5) - #2391 

Profile Page: 
- [x] Social icons links need an aria-label = #2545 
- [x] Increase contrast ratio for profile views count (current 3.56, required 4.5) - #2545 

Profile Links:
- [x] Modify UserLink component to use `link.icon` color only for the Icon. - #2424 

Docs Pages:
- [x] Add Title tags to all pages - #2496

**Comment by ManishBisht777** [comment]
Can I take up this issue? The Profile Page

**Comment by vedant-z** [comment]
@dan-mba May I work on Home page?

**Comment by dan-mba** [comment]
> @dan-mba May I work on Home page?

@vedant-z Go ahead, let me know if you need any help

**Comment by dan-mba** [comment]
> Can I take up this issue? The Profile Page

@ManishBisht777 I re-arranged some of the problems
The links contrast issues will probably require more discussion
If you want to work on the Profile Page social icons or any of the all pages problems let me know.

**Comment by ManishBisht777** [comment]
> > Can I take up this issue? The Profile Page
> 
> @ManishBisht777 I re-arranged some of the problems The links contrast issues will probably require more discussion If you want to work on the Profile Page social icons or any of the all pages problems let me know.

I want to work on socials icon first

**Comment by vedant-z** [comment]
@dan-mba I am not able to locate the home page.if you could point out to me it would be great.

**Comment by dan-mba** [comment]
> @dan-mba I am not able to locate the home page.if you could point out to me it would be great.

@vedant-z The home page code is in `pages/index.js`
The code for the 2 links is in a p tag that starts on [line 86](https://github.com/EddieHubCommunity/LinkFree/blob/main/pages/index.js#L86-L101)
Eddie is changing the users stats in #2391 , so that change is no longer relevant.

**Comment by dan-mba** [comment]
@eddiejaoude @EmmaDawsonDev 
Any thoughts on how to handle the contrast issues with the profile links?
My thought was to change the text o black while leaving the icon the current accent color.
It might also be nice to set the border to the accent or make some other use of it that doesn't affect the text contrast

**Comment by eddiejaoude** [comment]
I am happy with any improvements, we could keep tweaking it - I am not sure what the "correct" answer is though

**Comment by dan-mba** [comment]
> I am happy with any improvements, we could keep tweaking it - I am not sure what the "correct" answer is though

The question I was trying to get at was:
Does it make more sense to just change the text to black & design around that or try to fix all the colors to have a workable contrast?

**Comment by eddiejaoude** [comment]
Sorry for missing that. Hmm maybe black text would be the most straight forward?

**Comment by ManishBisht777** [comment]
https://webaim.org/resources/contrastchecker/

Maybe This should help

**Comment by CBID2** [comment]
Hi @dan-mba! I can try doing this task:
GitHub icon link in nav bar needs aria-label attribute
If it’s not taken of course

**Comment by dan-mba** [comment]
> Hi @dan-mba! I can try doing this task: GitHub icon link in nav bar needs aria-label attribute If it’s not taken of course

@CBID2 It is all yours. Let me know if you need any help.

**Comment by dan-mba** [comment]
@ManishBisht777 @vedant-z 
Any update on the issues you wanted to work on?
Let me know if you need any help

**Comment by CBID2** [comment]
> #2379

Hi @dan-mba! What line number in your code has the GitHub Icon? I'm having trouble finding it.

**Comment by dan-mba** [comment]
@CBID2 
I looked at your change and the aria-label needs double quotes on both sides.
`aria-label="GitHub Icon"`

Also, the PR you opened is against your fork & not the main LinkFree repo
In general, avoid line numbers in PR titles, save that for the description section.

**Comment by EmmaDawsonDev** [comment]
I can also chime in here a say that the word icon is not needed here. The screen reader itself will announce that it’s an image and a link so using icon in the label becomes too verbose.

**Comment by CBID2** [comment]
> @CBID2 
> 
> I looked at your change and the aria-label needs double quotes on both sides.
> 
> `aria-label="GitHub Icon"`
> 
> 
> 
> Also, the PR you opened is against your fork & not the main LinkFree repo
> 
> In general, avoid line numbers in PR titles, save that for the description section.
[Here you go](https://github.com/EddieHubCommunity/LinkFree/pull/2506)
